L1Trigger/L1THGCal/src/be_algorithms/HGCalMulticlusteringImpl.cc

```diff
@@ -81,32 +81,28 @@ void HGCalMulticlusteringImpl::clusterizeDR( const std::vector<edm::Ptr<l1t::HGC
 
     int iclu = 0;
     for(std::vector<edm::Ptr<l1t::HGCalCluster>>::const_iterator clu = clustersPtrs.begin(); clu != clustersPtrs.end(); ++clu, ++iclu){
-        
-        int imclu=0;
-        vector<int> tcPertinentMulticlusters;
-        for( const auto& mclu : multiclustersTmp ){
-            if( this->isPertinent(**clu, mclu, dr_) ){
-                tcPertinentMulticlusters.push_back(imclu);
-            }
-            ++imclu;
-        }
-        if( tcPertinentMulticlusters.empty() ){
-            multiclustersTmp.emplace_back( *clu );
-        }
-        else{
-            unsigned minDist = 1;
-            unsigned targetMulticlu = 0; 
-            for( int imclu : tcPertinentMulticlusters ){
-                double d = ( multiclustersTmp.at(imclu).centreProj() - (*clu)->centreProj() ).mag() ;
-                if( d < minDist ){
-                    minDist = d;
-                    targetMulticlu = imclu;
-                }
-            } 
 
-            multiclustersTmp.at( targetMulticlu ).addConstituent( *clu );
-            
-        }        
+      HGCalDetId cluDetId( (**clu).detId() );
```

Comment:
Indentations to be fixed. Keep things consistent with the original file: no tab, 4 spaces indent.